zerver/lib/onboarding.py

```diff
@@ -359,11 +355,11 @@ def send_initial_realm_messages(realm: Realm) -> None:
         )
 
         content1_of_greetings_topic_name = _("""
-This **greetings** topic is a great place to say "hi" :wave: to your teammates.
+This **greetings** topic is a great place to say “hi” :wave: to your teammates.
 """)
 
         content2_of_greetings_topic_name = _("""
-:point_right:  Click on any message to start a reply in the same topic.
+:point_right: Click on this message to start a new message in the same conversation.
```

Comment:
> Click on this message to start a new message in the same conversation

Could this be clearer like so?

> Click on this message to compose a message in this conversation.

Reply:
Maybe? I see where you're going, but somehow I'm finding it hard to parse.